.travis.yml

```diff
@@ -7,13 +7,12 @@ jobs:
 
   include:
     - stage: unit tests 👩🏽‍💻
-      script: npm run test:unit
+      script: npm run build && npm run test:unit
```

Comment:
since `npm test` already includes `npm run build` it is redundant here and below